docs/sql-ref-syntax-ddl-create-table-hiveformat.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: global
+title: CREATE HIVEFORMAT TABLE
+displayTitle: CREATE HIVEFORMAT TABLE
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+ 
+     http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+### Description
+
+The `CREATE TABLE` statement creates a new table using Hive format.
+
+### Syntax
+{% highlight sql %}
+CREATE [EXTERNAL] TABLE [IF NOT EXISTS] [db_name.]table_name
+  [(col_name1[:] col_type1 [COMMENT col_comment1], ...)]
+  [COMMENT table_comment]
+  [PARTITIONED BY (col_name2[:] col_type2 [COMMENT col_comment2], ...)]
+  [ROW FORMAT row_format]
+  [STORED AS file_format]
+  [LOCATION path]
+  [TBLPROPERTIES (key1=val1, key2=val2, ...)]
+  [AS select_statement]
+
+{% endhighlight %}
+
+### Parameters
+
+<dl>
+  <dt><code><em>EXTERNAL</em></code></dt>
+  <dd>Table is created using the path provided as LOCATION, does not use default location for this table.</dd>
+</dl>
+
+<dl>
+  <dt><code><em>PARTITIONED BY</em></code></dt>
+  <dd>Partitions are created on the table, based on the columns specified.</dd>
+</dl>
+
+<dl>
+  <dt><code><em>ROW FORMAT</em></code></dt>
+  <dd>SERDE is used to specify a custom SerDe or the DELIMITED clause inorder to use the native SerDe.</dd>
+</dl>
+
+<dl>
+  <dt><code><em>STORED</em></code></dt>
+  <dd>Table file format.</dd>
+</dl>
+
+<dl>
+  <dt><code><em>LOCATION</em></code></dt>
+  <dd>Specified Location is used to store table data.</dd>
+</dl>
+
+<dl>
+  <dt><code><em>COMMENT</em></code></dt>
+  <dd>Table comments are added.</dd>
+</dl>
+
+<dl>
+  <dt><code><em>TBLPROPERTIES</em></code></dt>
+  <dd>Table properties that has to be set are specified.</dd>
+</dl>
+
+<dl>
+  <dt><code><em>AS select_statement</em></code></dt>
+  <dd>The table is populated using the data from the select statement.</dd>
+</dl>
+
+
+### Examples
+{% highlight sql %}
+
+CREATE TABLE Student (Id INT,name STRING)
```

Comment:
This is not to create a Hive serde table. See https://github.com/apache/spark/pull/26736